client/package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Thai Pangsakulyanont <dtinth@spacet.me>",
   "license": "MIT",
   "scripts": {
-    "start": "DEV=1 webpack-dev-server --hot --port=9998 --host=0.0.0.0 --compress",
+    "start": "cross-env DEV=1 webpack-dev-server --hot --port=9998 --host=127.0.0.1 --compress",
```

Comment:
Cannot use `127.0.0.1` here, because I want the app to allow any host to connect (I usually run the dev server on my Mac and test it on my iPad).

I believe it’s possible to keep it as 0.0.0.0, but access it from `http://127.0.0.1:9998/`.

Reply:
Okay, will do.